trainingsanleitungen/scratch/scratch-fang-mich-v3.md

```diff
@@ -11,78 +11,64 @@ In diesem Spiel bist du ein Fisch, der dem großen Haifisch entkommen muss. Scha
-
-4. ![Fisch gruppieren](scratch-fang-mich-v3/04-to-group.png){: .right}
-Damit der kleine Fisch auch kleiner ist als der große Haifisch, müssen wir den Fisch verkleinern. Davor müssen wir ihn aber noch *gruppieren*, sodass im nächsten Schritt der ganze Fisch kleiner wird, nicht nur die Schwanzflosse. Wähle dazu den Fisch bei den Figuren aus, sodass er blau umrandet ist. Klicke anschließend auf den Tab *Kostüme*. Führe die Maus auf die karrierte Fläche und drücke die linke Maustaste. Halte diese gedrückt und bewege deine Maus nun so, dass sich der ganze Fisch innerhalb der grauen Linien befindet. Lass los und führe deine Maus nach oben, wo du rechts neben den beiden blauben Pfeilen ein Symbol siehst. Drücke darauf und der Fisch ist nun gruppiert.
-
-5. ![Fisch verkleinern](scratch-fang-mich-v3/05-resize.png){: .right}
-Nun kannst du den Fisch verkleinern, indem du ihn an den Ecken kleiner ziehst. *Tipp:* Wenn du den Fisch nur kleiner machen möchstest ohne ihn zu viel zu verändern, halte die Shift-Taste gedrückt, während du in kleiner ziehst.
+Jetzt brauchen wir einen Haifisch sowie einen Fisch, mit dem wir dem Haifisch entkommen wollen. Klicke dazu auf *Figur wählen* und füge einen Fisch sowie einen Haifisch dazu. Natürlich können es auch andere Figuren sein, zum Beispiel ein Käfer, der dem Vogel davonfliegt oder eine Person die dem Löwen davonläuft.
 
 7. ![Figuren benennen](scratch-fang-mich-v3/07-rename.png){: .right}
 Damit du später die Figuren leichter verwenden kannst, gib ihnen Namen wie *Hai* und *Fisch*. Du kannst den Namen von Figuren ändern, indem du unter der weißen Leinwand rechts neben dem Wort Figur auf das Namensfeld klickst und den neuen Namen eingibst.
 
 ## Fisch bewegen
 
 1. ![Fisch bewegen](scratch-fang-mich-v3/08-move-fish.png){: .right}
-Damit du den Fisch bewegen kannst, musst er nach links und rechts sowie oben und unten bewegt werden können.<br/><br/>
-  • Wähle zuerst den Fisch aus, damit er blau umrandet ist.<br/>
-  • Im Tab *Code* kannst du deinen Fisch nun bewegen. Verwende das Ereignis *Wenn Taste ... gedrückt* unter *Ereignisse*.<br/>
-  • Verknüpfe es jeweils mit einer Drehung *setze Richtung auf ...* unter *Bewegung*, damit der Fisch in die richtige Richtung schaut.<br/>
-  • Außerdem brauchen wir *gehe ...er Schritt*, um den Fisch zu bewegen.<br/><br/>
-Für *Pfeil nach oben* gedrückt: Richtung 0 Grad, gehe 10er-Schritte.<br/>
-Für *Pfeil nach unten* gedrückt: Richtung 180 Grad, gehe 10er-Schritte.<br/>
-Für *Pfeil nach rechts* gedrückt: Richtung 90 Grad, gehe 10er-Schritte.<br/>
-Für *Pfeil nach links* gedrückt: Richtung -90 Grad, gehe 10er-Schritte.<br/><br/>
+Damit du den Fisch bewegen kannst, musst er nach links und rechts sowie oben und unten bewegt werden können.
+  * Wähle zuerst den Fisch aus, damit er blau umrandet ist.
```

Comment:
Hier ist noch ein kleiner, genereller Fehler (gilt für alle Aufzählungen): Vor und nach Aufzählungen muss in Markdown immer eine Leerzeile sein. Tipp: Installiere dir [mardownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) in Visual Studio Code. Dann bekommst du Hinweise auf solche Fehler.